source/connect/required-access.txt

```diff
@@ -0,0 +1,97 @@
+.. _required-access:
+
+===============
+Required Access
+===============
+
+.. default-domain:: mongodb
+
+With authentication/authorization enforced on the connected MongoDB
+instance, the user requires appropriate privileges to access various
+features. The following table lists the privileges required to access
+the features as well as the built-in roles that can provide these
+privileges:
+
+.. note::
+   The built-in roles may provide more access than required. You can
+   also create a :ref:`user-defined-roles` to grant specific privileges.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 30 25 20
+
+   * - View
+     - Feature/Action
+     - Privilege(s)
+     - MongoDB Built-In Role
```

Comment:
Generally I'm having a hard time understanding this table.

Reply:
As I'm understanding it:

Compass View is the tab / view within Compass within which the action we're talking about lives.
Capability is the actual action we are discussing.
Privilege(s) is/are the privilege actions required to do perform said action.
Built-in Role is the name of the role which grants the required privileges.

Reply:
Got it, thanks.